src/nupic/utils/LogItem.cpp

```diff
@@ -66,7 +66,7 @@ LogItem::~LogItem()
   }
 
   if (ostream_ == nullptr)
-    ostream_ = &(std::cout);
+    ostream_ = &(std::cerr);
```

Comment:
This seems out of place in this PR. I don't have a good sense for what the right thing to do here is.